cfme/tests/intelligence/reports/test_widgets.py

```diff
@@ -76,7 +76,7 @@ def _finalize():
         with update(dashboard):
             dashboard.widgets = default_widgets
     request.addfinalizer(_finalize)
-    dashboard.reset_widgets()
+    reset_widgets()
```

Comment:
I have an issue with this function, it is not tied to an appliance object at all, what can we do about this?